2025-09-10 21:26:30,057 - werkzeug - WARNING -  * Debugger is active!
2025-09-10 21:26:30,057 - werkzeug - INFO -  * Debugger PIN: 307-682-953
2025-09-10 21:27:18,331 - werkzeug - INFO - 127.0.0.1 - - [10/Sep/2025 21:27:18] "GET / HTTP/1.1" 200 -
2025-09-10 21:27:32,744 - root - ERROR - Error occured in python script name [/media/shrav/New Volume/projects/ML_project/src/utils.py] line number [58] err message[[Errno 2] No such file or directory: 'artifacts/preprocessor.pkl']
2025-09-10 21:27:32,744 - root - ERROR - Error occured in python script name [/media/shrav/New Volume/projects/ML_project/src/utils.py] line number [62] err message[Error occured in python script name [/media/shrav/New Volume/projects/ML_project/src/utils.py] line number [58] err message[[Errno 2] No such file or directory: 'artifacts/preprocessor.pkl']]
2025-09-10 21:27:32,756 - werkzeug - INFO - 127.0.0.1 - - [10/Sep/2025 21:27:32] "[35m[1mPOST /predictdata HTTP/1.1[0m" 500 -
2025-09-10 21:27:32,778 - werkzeug - INFO - 127.0.0.1 - - [10/Sep/2025 21:27:32] "[36mGET /predictdata?__debugger__=yes&cmd=resource&f=style.css HTTP/1.1[0m" 304 -
2025-09-10 21:27:32,779 - werkzeug - INFO - 127.0.0.1 - - [10/Sep/2025 21:27:32] "[36mGET /predictdata?__debugger__=yes&cmd=resource&f=debugger.js HTTP/1.1[0m" 304 -
2025-09-10 21:27:32,786 - werkzeug - INFO - 127.0.0.1 - - [10/Sep/2025 21:27:32] "GET /predictdata?__debugger__=yes&cmd=resource&f=console.png&s=rTPgzE6b6MujIMoroOpG HTTP/1.1" 200 -
2025-09-10 21:27:32,808 - werkzeug - INFO - 127.0.0.1 - - [10/Sep/2025 21:27:32] "[36mGET /predictdata?__debugger__=yes&cmd=resource&f=console.png HTTP/1.1[0m" 304 -
2025-09-10 21:28:22,711 - werkzeug - INFO -  * Detected change in '/media/shrav/New Volume/projects/ML_project/src/pipeline/predict_pipeline.py', reloading
